conans/test/integration/package_info_test.py

```diff
@@ -46,3 +52,184 @@ def package_info(self):
+                        self.output.info("COMP %s Binary paths: %s" % (comp_name, comp_value.bin_paths))
+                        self.output.info("COMP %s Lib: %s" % (comp_name, comp_value.lib))
+                        self.output.info("COMP %s Exe: %s" % (comp_name, comp_value.exe))
+                        self.output.info("COMP %s Deps: %s" % (comp_name, comp_value.deps))
+        """)
+
+        client = TestClient()
+        client.save({"conanfile_dep.py": dep, "conanfile_consumer.py": consumer,
+                     "galaxy/starlight/starlight.h": "",
+                     "lib/libstarlight": "",
+                     "galaxy/planet/planet.h": "",
+                     "lib/libplanet": "",
+                     "galaxy/iss/iss.h": "",
+                     "iss_libs/libiss": "",
+                     "bin/exelauncher": ""})
+        dep_ref = ConanFileReference("dep", "1.0", "us", "ch")
+        dep_pref = PackageReference(dep_ref, NO_SETTINGS_PACKAGE_ID)
+        client.run("create conanfile_dep.py dep/1.0@us/ch")
+        client.run("create conanfile_consumer.py consumer/1.0@us/ch")
+        package_folder = client.cache.package_layout(dep_ref).package(dep_pref)
+
+        expected_global_include_paths = [os.path.join(package_folder, "galaxy", "starlight"),
+                                         os.path.join(package_folder, "galaxy", "planet"),
+                                         os.path.join(package_folder, "galaxy", "iss")]
+        expected_global_library_paths = [os.path.join(package_folder, "lib"),
+                                         os.path.join(package_folder, "iss_libs")]
+        expected_global_binary_paths = [os.path.join(package_folder, "bin")]
+        expected_global_libs = ["libstarlight", "ground", "libplanet", "solar", "magnetism", "libiss"]
+        expected_global_exes = ["exelauncher"]
+        fromatted1 = "GLOBAL Include Paths: %s" % expected_global_include_paths
+        self.assertIn(fromatted1, client.out)
+        self.assertIn("GLOBAL Library Paths: %s" % expected_global_library_paths, client.out)
+        self.assertIn("GLOBAL Binary Paths: %s" % expected_global_binary_paths, client.out)
+        self.assertIn("GLOBAL Libs: %s" % expected_global_libs, client.out)
+        self.assertIn("GLOBAL Exes: %s" % expected_global_exes, client.out)
+
+        formatted = "GLOBAL Include Paths: {}".format(expected_global_include_paths)
+        print(formatted)
+        self.assertIn(formatted, client.out)
+        self.assertIn("DEPS Library Paths: %s" % expected_global_library_paths, client.out)
+        self.assertIn("DEPS Binary Paths: %s" % expected_global_binary_paths, client.out)
+        self.assertIn("DEPS Libs: %s" % expected_global_libs, client.out)
+        self.assertIn("DEPS Exes: %s" % expected_global_exes, client.out)
+
+        #TODO: Complete the test checking the output of the components
```

Comment:
:goberserk: go!

Reply:
Cannot complete until I figure out why the asserts above are failing in one place and not in the step before 😆

Reply:
ok, I'll have a look at those assets too

Reply:
🙃